galaxy_skeleton2.R

```diff
@@ -79,7 +83,7 @@ variable_names <-
 # super for this kind of search-and-replace. Replace the question mark below.
 
 comma_separated_values <- 
-  ? %>% 
+  raw_file %>% 
```

Comment:
Here, you should have specified an index of the raw file. Not doing so results in including both descritions and column names with the seperator line. This results in the data frame not being tidy and correct later in the code. 

To index the data, you can write raw_file[(L+1):length(raw_file)]